src/sregulator.erl

```diff
@@ -411,7 +411,6 @@ dirty_cancel(Regulator, Tag) ->
       Regulator :: regulator(),
       Ref :: reference(),
       Go :: {go, Ref, Pid, RelativeTime, SojournTime},
-      Ref :: reference(),
```

Comment:
Otherwise we get an Erlang/OTP `edoc` -related exception (even when using the new plugin, which uses `edoc` behind the scenes).